project/frontend/src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import classes from "./Dashboard.module.css";
+import classnames from "classnames";
+import LinkButton from "../../components/UI/LinkButton/LinkButton";
+import Block from "../../components/UI/Block/Block";
+import ShortSGonksList from "../../components/SGonksLists/ShortSGonksList/ShortSGonksList";
+import TrendingSearches from "../../components/TrendingSearches/TrendingSearches";
+import RecentBuys from "../../components/RecentBuys/RecentBuys";
+
+const Dashboard = (props) => {
+  const placeholderSGonks = [
+    {
+      searchTerm: "chicken wings",
+      currentPrice: "39122",
+      priceYesterday: "29319",
+    },
+    {
+      searchTerm: "chicken nuggets",
+      currentPrice: "4214",
+      priceYesterday: "321442",
+    },
+    {
+      searchTerm: "random term",
+      currentPrice: "321",
+      priceYesterday: "4155",
+    },
+    {
+      searchTerm: "spite zero",
+      currentPrice: "2313",
+      priceYesterday: "2313",
+    },
+    {
+      searchTerm: "escher",
+      currentPrice: "321",
+      priceYesterday: "654",
+    },
+    {
+      searchTerm: "react donut",
+      currentPrice: "32145",
+      priceYesterday: "532578",
+    },
+    {
+      searchTerm: "asdfdsagsafwg qweqrqewqe",
+      currentPrice: "61512",
+      priceYesterday: "76522",
+    },
+    {
+      searchTerm: "unicorns",
+      currentPrice: "321809",
+      priceYesterday: "321321",
+    },
+    {
+      searchTerm: "eek",
+      currentPrice: "321",
+      priceYesterday: "321521",
+    },
+  ];
+
+  const placeholderTrends = [
+    "Kellyanne Conway",
+    "Filibuster",
+    "Fultondale al",
+    "Ben Askren",
+    "Stock-market",
+    "Etsy stock",
+    "Elliot Page",
+    "Susan Rice",
+    "Raya and the Last Dragon",
+    "Starlink",
+  ];
+
+  const placeholderRecentBuys = [
```

Comment:
how will recent buys update? Page load time OR will recent buys made after page has loaded be pushed to the web browser?

Reply:
Recent buys is not updated within one session, it just stays the same. But if the user somehow knows that there are new recent buys, and wants to look at them, they can refresh, and the data will be refetched.